### PR TITLE
Included makefile and packaging resources for build

### PR DESCRIPTION
**Pull request by bgphipps**
@sheeley @mgloystein Quick and easy makefile for installing resources and combining lua files. 

We can/should include tests in the `Makefile` when we're at that point. 

`pack.lua` was a small snippet I found for combining files. I'm open to `squish` ( @mgloystein suggested it) if we feel we need more capabilities rather than just combining files.

### CODE REVIEW

**Comment by mgloystein** [comment]
Coolio

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+planout:
+	lua buildresources/pack.lua src/ > dist/planout.lua
+
+install:
+	brew install lua
```
Comment (sheeley) [inline Makefile]: Might want to note that this is OS X specific, and/or support other things? Also would be useful to include a README.md explaining how this (and other things) work.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	luarocks install penlight
+
+clean:
+	rm dist/planout.lua
```
Comment (sheeley) [inline Makefile]: Adding `-f` will keep this from complaining if the file doesn't exist.

File: buildresources/pack.lua [hunk 3]
```diff
@@ -0,0 +1,71 @@
+-- pack.lua
+-- released under the Romantic WTF Public License
+-- http://getmoai.com/wiki/index.php?title=User:Pygy/Romantic_WTF_Public_License
```
Comment (sheeley) [inline buildresources/pack.lua:3]: great license

Comment (bgphipps) [inline buildresources/pack.lua:3]: yeah, it's pretty silly.

File: Makefile (hunk 2 above)
Comment (bgphipps) [inline Makefile]: I'll update it.

File: Makefile (hunk 1 above)
Comment (mgloystein) [inline Makefile]: Any version of make should work.

**Comment by bgphipps** [comment]
@sheeley @mgloystein per our conversation, I removed `dist/planout.lua` and removed installing `lua` in `make install`

I'll work on circleci integration in a different pr.

**Pull request merged**